### DISCUSSION TITLE
jekyll reverts new favicon to default favicon

### DISCUSSION THREAD

**Discussion by bL34cHig0** [cotes2020/jekyll-theme-chirpy#1434] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Hey y'all,

I tried to customize the `favicon` on my jekyll-chirpy site, and I followed the [guide](https://chirpy.cotes.page/posts/customize-the-favicon/) on cotes's page. 

However, everytime I replace all `*.PNG` and `*.ICO` files in **assets/img/favicons/** with the new ones generated by `Favicon generator`, the favicon reverts back to the default one instead of the new one I uploaded after previewing my changes on **localhost**

I use this command to preview my changes on localhost: `bundle exec jekyll s` 

Note that the image I used on `Favicon generator` is a square image and after downloading the generated packages, I deleted `browserconfig.xml` and `site.webmanifest` from the extracted folder. Which I did not add to  **assets/img/favicons/**.

Please, any advice on how to resolve this issue will be greatly appreciated. Thank you.

P.S my local machine is Windows

### Operations you have already tried

I used other images but still had the same issue.

**Comment by chuckhoupt** [comment]
Have you tried clearing your browser's cache? Facvicons can be very "sticky" in the cache.

**Reply by bL34cHig0** [reply]
Thank you for your response. I cleared my browser's cache and the issue still persists.

**Reply by chuckhoupt** [reply, marked as answer]
It looks like you're putting your new icon in `_site`, which is the directory where Jekyll puts the built site (i.e. every time Jekyll builds, it completely overwrites the contents of `_site`). If you place the icons in `example.github.io/assets/img/favicons` (rather than `example.github.io/_site/assets/...`), then they should work.

**Reply by bL34cHig0** [reply]
Thanks for pointing that out to me. However, I added the icons to `myusername.github.io/assets/img/favicons` and it still won't work. Although there is another folder called `lib` in the `assets` folder.

**Reply by bL34cHig0** [reply]
Oh it worked. Thank you for your assistance.
